lib/manageiq/providers/workflows/engine.rb

```diff
@@ -33,6 +33,25 @@ def self.apply_logger_config(config)
         def self.seedable_classes
           %w[ManageIQ::Providers::Workflows]
         end
+
+        def self.floe_docker_runner
+          require "miq_environment"
```

Comment:
Super-duper minor, and I won't hold up merge, but I think technically this needs a `require "floe"`, since the method uses it.  I know it's called in the initializer first anyway, so not a big deal.

Reply:
Added